src/renderers/WebGLRenderer.js

```diff
@@ -2812,14 +2814,16 @@ function WebGLRenderer( parameters ) {
 
 		if ( srcTexture.isDataTexture ) {
 
-			_gl.texSubImage2D( _gl.TEXTURE_2D, level || 0, position.x, position.y, width, height, glFormat, glType, srcTexture.image.data );
+			_gl.texSubImage2D( _gl.TEXTURE_2D, level, position.x, position.y, width, height, glFormat, glType, srcTexture.image.data );
 
 		} else {
 
-			_gl.texSubImage2D( _gl.TEXTURE_2D, level || 0, position.x, position.y, glFormat, glType, srcTexture.image );
+			_gl.texSubImage2D( _gl.TEXTURE_2D, level, position.x, position.y, glFormat, glType, srcTexture.image );
 
 		}
 
+		if ( level == 0 && dstTexture.generateMipmaps ) _gl.generateMipmap( _gl.TEXTURE_2D );
```

Comment:
I am confused now...

Is this the intention?

```js
if ( level === undefined && dstTexture.generateMipmaps ) _gl.generateMipmap( _gl.TEXTURE_2D );
```

Reply:
No... the intention is only to autogenerate mipmaps if the developer is updating the level 0 texture, and when the generateMipmaps flag is set.

Reply:
Ah! Okay.

Do you mind adding a comment?

```js
// Generate mipmaps only when copying level 0
```

Reply:
Added